config/zones.json

```diff
@@ -1601,19 +1601,19 @@
     ],
     "capacity": {
       "battery storage": 7,
-      "biomass": 1579,
+      "biomass": 2234,
       "coal": 1818,
-      "gas": 12218,
+      "gas": 12752,
       "geothermal": 16,
       "hydro": 18743.6955,
       "hydro storage": 5029.87,
       "nuclear": 61370,
-      "oil": 2746,
-      "solar": 12447,
+      "oil": 2860,
```

Comment:
The source seems to have a value of 2890 MW for oil.

Reply:
Yes indeed, 2890 MW is the correct value.

Reply:
Change made here too.